mantidproject/mantid
# Show keyboard shortcuts in Workbench editor's Options menu

**Description of work.**
It seems Qt 5.10 decided that all QMenus not inside a QMenuBar are context menus, relevant Qt change: https://codereview.qt-project.org/#/c/219931/.

This PR makes the keyboard shortcuts visible inside the Options menu on the Workbench editor again.

**To test:**
Open Workbench, click the `Options` menu, shortcuts should be displayed inside



Fixes #24993




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
